sky/clouds/service_catalog/data_fetchers/fetch_aws.py

```diff
@@ -268,27 +268,30 @@ def get_all_regions_instance_types_df(regions: Set[str]) -> pd.DataFrame:
 _UBUNTU_VERSION = ['18.04', '20.04']
 
 
-def _get_image_id(region: str, ubuntu_version: str, creation_date: str) -> str:
+def _get_image_id(region: str, ubuntu_version: str,
```

Comment:
While we're refactoring, can we also change the name of this method (perhaps to `_fetch_image_id` or something else)? Reason being it collides with `clouds.aws._get_image_id` and makes search in the codebase a little confusing.